ibm/resource_ibm_database.go

```diff
@@ -1511,7 +1511,7 @@ func resourceIBMDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 		if err != nil {
 			return fmt.Errorf("Error updating database scaling group: %s", err)
 		}
-		_, err = waitForDatabaseTaskComplete(task.Id, d, meta)
+		_, err = waitForDatabaseTaskComplete(task.Id, d, meta, d.Timeout(schema.TimeoutUpdate))
 		if err != nil {
 			return fmt.Errorf(
 				"Error waiting for database (%s) scaling group update task to complete: %s", icdId, err)
```

Comment:
`"Error waiting for database (%s) scaling group update task to complete: %s"` ==> `"Error waiting for database (%s) auto scaling group memory update task to complete: %s"`